lightning/src/chain/keysinterface.rs

```diff
@@ -273,8 +276,8 @@ pub struct InMemoryChannelKeys {
 	funding_key: SecretKey,
 	/// Local secret key for blinded revocation pubkey
 	revocation_base_key: SecretKey,
-	/// Local secret key used in commitment tx htlc outputs
-	payment_base_key: SecretKey,
+	/// Local secret key used in commitment txn generated by us (for broadcast by our counterparty)
```

Comment:
Unclear, "local secret key generated by us, used in remote commitment transaction" ?